cmd/cli/cmd.go

```diff
@@ -316,6 +316,9 @@ func NewCommand() (*cobra.Command, *int) {
 	daemonCommand.PersistentFlags().StringP("clean", "c", config.ViperConfig.GetString("clean"), fmt.Sprintf("clean options before %s: %s", setupCommand.Name(), options.GetOptionNames(options.Clean{})))
 	config.ViperConfig.BindPFlag("clean", daemonCommand.PersistentFlags().Lookup("clean"))
 
+	daemonCommand.PersistentFlags().StringP("keep", "k", config.ViperConfig.GetString("keep"), fmt.Sprintf("clean everything but the given options before %s: %s", setupCommand.Name(), options.GetOptionNames(options.Clean{})))
```

Comment:
What is the behaviour of `keep` if the setup command has never been called? Just a no-op?

Reply:
It's always called by `clean`.

As the workflow is `clean` -> `setup` -> `run`.